### ISSUE TITLE
Resizing doesn't move the label to stay at the bottom of the icon

### ISSUE THREAD

**Issue by SvenAelterman** [David-Summers/Azure-Design#33]
When resizing the shapes, the label doesn't adjust its position:
![image](https://user-images.githubusercontent.com/17446043/119005345-5c813200-b95d-11eb-8178-522d6ec9be24.png)

**Comment by David-Summers** [comment]
Hi Sven

Thank you for finding this and your support!!!

I found the issue here, a few versions ago I changed the way I was positioning the Text box. Initially it was defined as the Height of the Icon * -0.3 so it was positioned relative to the center of the icon. I changed this to a fixed value of -14mm so that the text field lined up for all icons regardless of the icon width or height. Turns out this is the cause of your re-sizing issue. 

Working out a way to do this mathematically to avoid this.

![image](https://user-images.githubusercontent.com/53208734/119060943-8fd3c780-ba16-11eb-9817-7a84f9ff4502.png)

Just need to update my script to get the current height and adjust the ratio...
